hbase/hbase.sh

```diff
@@ -54,20 +59,66 @@ function install_and_configure_hbase() {
   </configuration>
 EOF
 
+  cat << EOF > /etc/systemd/system/hbase-master.service
+[Unit]
+Description=HBase Master
+Wants=network-online.target
+After=network-online.target hadoop-hdfs-namenode.service
+
+[Service]
+User=root
+Group=root
+Type=simple
+EnvironmentFile=/etc/environment
+Environment=HBASE_HOME=/etc/hbase
+ExecStart=/usr/bin/hbase \
+  --config ${HBASE_HOME}/conf/ \
+  master start
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+  cat << EOF > /etc/systemd/system/hbase-regionserver.service
+[Unit]
+Description=HBase Regionserver
+Wants=network-online.target
+After=network-online.target hadoop-hdfs-datanode.service
+
+[Service]
+User=root
+Group=root
+Type=simple
+EnvironmentFile=/etc/environment
+Environment=HBASE_HOME=/etc/hbase
+ExecStart=/usr/bin/hbase \
+  --config ${HBASE_HOME}/conf/ \
+  regionserver start
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+  systemctl daemon-reload
+
+  if [[ -z "${hbase_root_dir}" ]]; then
+    hbase_root_dir="hdfs://${CLUSTER_NAME}-m-0:8020/hbase"
+  fi
+  bdconfig set_property \
+    --configuration_file 'hbase-site.xml.tmp' \
+    --name 'hbase.rootdir' --value "${hbase_root_dir}" \
+    --clobber
   if [[ "${MASTER_ADDITIONAL}" != "" ]]; then
     # If true than init action is running on high availability dataproc cluster.
     # HBase will use zookeeper service pre-installed and running on master nodes.
     bdconfig set_property \
       --configuration_file 'hbase-site.xml.tmp' \
       --name 'hbase.zookeeper.quorum' --value "${zookeeper_nodes}-0,${MASTER_ADDITIONAL}" \
       --clobber
-    bdconfig set_property \
-      --configuration_file 'hbase-site.xml.tmp' \
-      --name 'hbase.rootdir' --value "hdfs://${CLUSTER_NAME}-m-0:8020/hbase" \
```

Comment:
Removing this will cause HBase master being unable to start on standard and single node clusters. I will refactor this part of code to be more readable.